src/components/input/select/index.tsx

```diff
@@ -293,7 +292,7 @@ export class SmoothlyInputSelect implements Input, Editable, Clearable, Componen
 				</div>
 				<div class="icons" ref={element => (this.iconsDiv = element)}>
 					<slot name="end" />
-					{this.showArrow && (
+					{this.looks == "border" && (
```

Comment:
Are we sure we only want arrows with this look? With the previous prop it was controllable for any look.

Reply:
Yes, that was the requested change by Simon.